### PR TITLE
Add additional details for how community members can participate, including design reviews

### PR DESCRIPTION
**Pull request by mrjoro**
Updates CONTRIBUTING.md to make it more clear how community members can participate in the community, including significant on a new design review process.

cc @pbakaus @bpaduch @rudygalfi @adelinamart

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,6 +32,20 @@ As with all pieces of software, you may end up running into bugs. Please submit
...
+* **Weekly design reviews** are held as video conferences via Google Hangouts on Wednesdays at [1pm Pacific](https://www.google.com/?#q=1pm+pacific+in+local+time).  Design reviews are used to discuss/refine engineering designs after an initial draft of the design has been created and shared with the community.
+  * We use GitHub issues labeled [Type: Design Review](https://github.com/ampproject/amphtml/issues?q=label%3A%22Type%3A+Design+Review%22) to track design reviews.  The Design Review issue for a given week will have a link to the design docs being discussed that week as well as a link to the Hangout.
+  * When you attend a design review please read through the design docs before the review starts.
+  * If you have an engineering design you would like to discuss at a design review:
+    * Document your design in a shared Google Document open to public comments. Take a look at [Design docs - A design doc](https://medium.com/@cramforce/design-docs-a-design-doc-a152f4484c6b) for tips on putting together a good design doc.
```
Comment (cramforce) [inline CONTRIBUTING.md]: Add something like "These docs do not have to be very long. Here is a list of examples we've discussed in the past"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,6 +16,10 @@ limitations under the License.
 
 ## Contributing to AMP HTML
 
+The AMP HTML project strongly encourages technical contributions!
```
Comment (cramforce) [inline CONTRIBUTING.md]: Do you think it makes sense to link to https://www.ampproject.org/contribute/ or would that we redundant?

Comment (mrjoro) [inline CONTRIBUTING.md]: Makes sense to me; they'll refer to each other now so people who start at one will know about the other.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mrjoro) [inline CONTRIBUTING.md]: I've added a comment about short docs being okay and filed Issue #7281 for tracking adding some example design docs when we have some.

(I also added a line to the pre-review section indicating it's fine to request a pre-review before the doc is complete.)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -28,6 +32,20 @@ As with all pieces of software, you may end up running into bugs. Please submit
...
+* **Weekly status updates** from individual community members are posted as GitHub issues labeled [Meeting Notes](https://github.com/ampproject/amphtml/issues?q=label%3A%22Meeting+Notes%22).  If you have a weekly status update related to your work on AMP that you'd like to share with the community please add it as a comment on the relevant Meeting Notes issue.
+* **Weekly design reviews** are held as video conferences via Google Hangouts on Wednesdays at [1pm Pacific](https://www.google.com/?#q=1pm+pacific+in+local+time).  Design reviews are used to discuss/refine engineering designs after an initial draft of the design has been created and shared with the community.
+  * We use GitHub issues labeled [Type: Design Review](https://github.com/ampproject/amphtml/issues?q=label%3A%22Type%3A+Design+Review%22) to track design reviews.  The Design Review issue for a given week will have a link to the design docs being discussed that week as well as a link to the Hangout.
+  * When you attend a design review please read through the design docs before the review starts.
```
Comment (cramforce) [inline CONTRIBUTING.md]: For follow up: @aghassemi and I discussed that we should make this language a little less scary in terms of: "Contributing often doesn't require a design and certainly doesn't require an in-person meeting, but feel free to join us if that helps!"

Comment (mrjoro) [inline CONTRIBUTING.md]: SG; I added this to issue #7285 to track it.

**Pull request merged**